store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9929,4 +9929,10 @@ TODO: delete them permanently from here
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+<attr id="5000" name="zimbraGlobalExternalStoreConfig" type="string" cardinality="single" optionalIn="globalConfig" since="9.1.0">
```

Comment:
since there can be multiple configs, I think we  should make multi value attribute

Reply:
Updated cardinality="multi"

Reply:
What is here "multiple config" - we are storing info in one json object

@RakeshAMore - do you want to store multiple json object and if yes please tell one test case?

Reply:
This atrr should be single value, reverted to siungle cardinality